src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/AdministrationController.php

```diff
@@ -184,4 +195,105 @@ protected function getNotificationsFormHandler(): FormHandlerInterface
     {
         return $this->get('prestashop.adapter.administration.notifications.form_handler');
     }
+
+    /**
+     * @var DataProviderErrorCollection
+     */
+    private function getErrorMessages(DataProviderErrorCollection $errors): array
+    {
+        $messages = [];
+
+        foreach ($errors as $error) {
+            $messages[] = $this->getErrorMessage($error);
+        }
+
+        return $messages;
+    }
+
+    /**
+     * @param DataProviderError $error
+     *
+     * @return string
+     */
+    private function getErrorMessage(DataProviderError $error): string
+    {
+        switch ($error->getErrorCode()) {
+            case FormDataProvider::ERROR_NOT_NUMERIC_OR_LOWER_THEN_ZERO:
+                return $this->trans(
+                    '%s is invalid. Please enter an integer greater or equal to 0.',
+                    'Admin.Notifications.Error',
+                    [$this->getFieldLabel($error->getFieldName())]
+                );
+            case FormDataProvider::ERROR_COOKIE_LIFETIME_MAX_NUMBER_EXCEEDED:
+                return $this->trans(
+                    '%s is invalid. Please enter number lower then %s.',
```

Comment:
```suggestion
                    '%s is invalid. Please enter a number lower than %s.',
```

Does this number have to be an integer? If yes, maybe we should replace "number" with "integer" for more consistency. Wdyt?

Reply:
Yes it's an integer, i changeed wording to
  `` %s is invalid. Please enter an integer lower than %s.``
  Thanks for suggestion!